web/src/modules/feedback/FeedbackInformation.js

```diff
@@ -0,0 +1,26 @@
+// @flow
```

Comment:
What is the difference to FeedbackVariant/Why do we need both?

Reply:
aactually something must have happened here. I thought I replaved this FeedbackInformation file with the FeedbackVariant again. But to explain the current structure: 
The Pages, Categories, Offers, ... etc navigate to the FeedbackModal with the FeedbackInformation. There the feedbackVariants are created depending on the properties of the FeedbackInformation. And finally the FeedbackParams are created when a specific feedbackVariant was selected and submitted.

Reply:
So this has to be changed? Or is this correct/necessary now? Not sure if I understand the first part of your explanation correctly. Thanks!

Reply:
Ahh. I accidentally created the class in web. That's why I didn't find it ^^ 
Removed as not used.